apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.IntegrationTests/SimplePubSubTest.cs

```diff
@@ -44,16 +44,21 @@ public SimplePubSubTest(PubsubFixture fixture)
 
         private async Task RunBulkMessaging(
             int messageCount, int minMessageSize, int maxMessageSize, int maxMessagesInFlight, int initialNackCount,
-            TimeSpan? timeouts = null, int? cancelAfterRecvCount = null)
+            TimeSpan? timeouts = null, int? cancelAfterRecvCount = null, TimeSpan? interPublishDelay = null,
+            TimeSpan? debugOutputPeriod = null)
```

Comment:
Our *three* main TimeSpans are timeouts, interPublishDelay, and debugOutputPeriod... (I'll come in again...)

Reply:
:)